content/docs/instrumenting/exporters.md

```diff
@@ -10,42 +10,28 @@ metrics from third-party systems as Prometheus metrics. This is useful for
 cases where it is not feasible to instrument a given system with Prometheus
 metrics directly (for example, HAProxy or Linux system stats).
 
-## Official third-party exporters
-
-These exporters are maintained as part of the official
-[Prometheus GitHub organization](https://github.com/prometheus):
-
-   * [Node/system metrics exporter](https://github.com/prometheus/node_exporter)
-   * [AWS CloudWatch exporter](https://github.com/prometheus/cloudwatch_exporter)
-   * [Blackbox exporter](https://github.com/prometheus/blackbox_exporter)
-   * [Collectd exporter](https://github.com/prometheus/collectd_exporter)
-   * [Consul exporter](https://github.com/prometheus/consul_exporter)
-   * [Graphite exporter](https://github.com/prometheus/graphite_exporter)
-   * [HAProxy exporter](https://github.com/prometheus/haproxy_exporter)
-   * [InfluxDB exporter](https://github.com/prometheus/influxdb_exporter)
-   * [JMX exporter](https://github.com/prometheus/jmx_exporter)
-   * [Memcached exporter](https://github.com/prometheus/memcached_exporter)
-   * [Mesos task exporter](https://github.com/prometheus/mesos_exporter)
-   * [MySQL server exporter](https://github.com/prometheus/mysqld_exporter)
-   * [SNMP exporter](https://github.com/prometheus/snmp_exporter)
-   * [StatsD exporter](https://github.com/prometheus/statsd_exporter)
+## Third-party exporters
 
-The [JMX exporter](https://github.com/prometheus/jmx_exporter) can export from a
-wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.org/) and
-[Cassandra](http://cassandra.apache.org/).
+Some of these exporters are maintained as part of the official [Prometheus GitHub organization](https://github.com/prometheus),
+those are marked as *official*, others are externally contributed and maintained.
 
-## Independently maintained third-party exporters
-
-There are also a number of exporters which are externally contributed
-and maintained. We encourage the creation of more exporters but cannot
+We encourage the creation of more exporters but cannot
 vet all of them for best practices. Commonly, those exporters are
```

Comment:
Done. [This one](https://prometheus.io/docs/instrumenting/writing_exporters/) you ment? Just that we avoid miscommunication here.